### PR TITLE
187121501 currency conv new conversion 

### PR DESCRIPTION
**Pull request by tiffanylamm**
### [Pivotal Tracker Link][tracker]

[tracker]: https://www.pivotaltracker.com/story/show/187121501

## What this PR does:

This pull request fixes the view implemented for admins inputting new currency conversions. The from_currency is now based on the currency of a conference's ticket and to_currency is based on  a drop down list of a hard coded list from the currency_conversion model. Also fixes issues with migrating from submission instructions to submission_template.

#### Who authored this PR?
@tiffanylamm 

### How should this PR be tested?
There are written tests to test the feature and it's flow as well as edge cases. There is also validation testing for user input. There are added tests for correct erroring for cases like when a conference doesn't have any tickets.

#### Are there any complications to deploying this?

No.

### Checklist:

- [x] Has this been deployed to a staging environment or reviewed by a customer?
- [x] Tag someone for code review (either a coach / team member)
- [ ] I have renamed the branch to match PivotTracker's suggested one (necessary for BlueJay)

### CODE REVIEW

File: app/models/currency_conversion.rb [hunk 1]
```diff
@@ -4,12 +4,24 @@
...
+#
+#  index_currency_conversions_on_conference_id  (conference_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (conference_id => conferences.id)
 #
 class CurrencyConversion < ApplicationRecord
+  VALID_CURRENCIES = %w[AUD CAD CHF CNY EUR GBP JPY USD].freeze
   belongs_to :conference
```
Comment (fp456) [inline app/models/currency_conversion.rb:25]: Are these the only currencies that will be available? If not, it may be more convenient to use a library to not hardcode all the currencies available

Comment (fp456) [reply app/models/currency_conversion.rb:25]: ignore comment, clarification was done

**Review by fp456 — APPROVED** [review]

File: Gemfile.lock [hunk 2]
```diff
@@ -380,6 +380,9 @@ GEM
     nio4r (2.7.0)
     nokogiri (1.16.3-arm64-darwin)
       racc (~> 1.4)
+    nokogiri (1.16.2-x86_64-darwin)
+      racc (~> 1.4)
+    nokogiri (1.16.2-x86_64-linux)
```
Comment (cycomachead) [inline Gemfile.lock:385]:
```suggestion
```


oops, my bad. these lines should not be here.

**Review by cycomachead — APPROVED** [review]
🎉 thanks! looks great!
